### PR TITLE
Add support for InterleavedBufferAttribute to Box3 and Mesh.

### PR DESCRIPTION
**Pull request by indif**
Add support for InterleavedBufferAttribute to Box3 and Mesh.

### CODE REVIEW

File: src/objects/Mesh.js [hunk 1]
```diff
@@ -208,11 +208,43 @@ THREE.Mesh.prototype = Object.assign( Object.create( THREE.Object3D.prototype ),
 				var a, b, c;
 				var index = geometry.index;
 				var attributes = geometry.attributes;
-				var positions = attributes.position.array;
+				var positions;
+				if ( attributes.position instanceof THREE.InterleavedBufferAttribute ) {
+
+					positions = {
```
Comment (mrdoob) [inline src/objects/Mesh.js]: Is there a way to avoid creating these objects? Maybe refactoring something from outside?

**Comment by sc0rp1d** [comment]
https://github.com/mrdoob/three.js/blob/master/src/core/BufferGeometry.js#L109
https://github.com/mrdoob/three.js/blob/master/src/core/BufferGeometry.js#L551
https://github.com/mrdoob/three.js/blob/master/src/core/BufferGeometry.js#L579
https://github.com/mrdoob/three.js/blob/master/src/core/BufferGeometry.js#L634
https://github.com/mrdoob/three.js/blob/master/src/core/BufferGeometry.js#L797

Also don't support InterleavedBufferAttribute

**Comment by mrdoob** [comment]
Maybe we should move all this code to `BufferAttribute` and `InterleavedBufferAttribute` instead? 

```js
scope.copy( geometry.attributes.position.computeBoundingSphere( node.matrixWorld ) );
```

/cc @WestLangley

**Comment by indif** [comment]
rewrite in #9242

**Pull request closed without merging**